phongo_version.h

```diff
@@ -22,9 +22,9 @@
  * publishing a release. */
 
 /* clang-format off */
-#define PHP_MONGODB_VERSION "1.18.1-dev"
+#define PHP_MONGODB_VERSION "1.18.2-dev"
```

Comment:
We discussed this earlier, but to repeat here: I think the process would benefit from splitting the changes up into separate PRs. That would presumably not require any special merging if automation were to create the tag after the PR is merged (in any fashion). And it would also make reviewing the diff(s) more straightforward since the most meaningful change (the "Packed 1.18.1" commit) would not longer be hidden.